README.md

```diff
@@ -341,6 +341,7 @@ Table of Contents
   * [flock.com](https://flock.com) — A faster way for your team to communicate. Free Unlimited Messages, Channels, Users, Apps & Integrations
   * [Flowdash](https://flowdash.com/) — Automate business workflows. Free tier for up to 3 users, 1000 tasks, and 3 workflows.
   * [flowdock.com](https://www.flowdock.com/) — Chat and inbox, free for teams up to 5
+  * [Gather](https://www.gather.town/) - A better way to meet online. Centered around fully customizable spaces, Gather makes spending time with your communities just as easy as real life.
```

Comment:
maybe add something like "Free for 25 concurrent users"

Reply:
Done. @ripienaar